pyproject.toml

```diff
@@ -54,3 +57,66 @@ strict_concatenate = true
 [tool.bandit]
 tests = ["B402","B401","B310","B403","B404","B302","B321","B301","B312","B303","B306","B304","B305","B308","B307","B313","B320","B405","B409","B408","B407","B406","B410","B411","B101","B102","B201","B104","B107","B106","B105","B608","B108","B701","B609","B601","B501","B103","B507","B502","B504","B605","B606","B607","B602","B603","B112","B110","B702","B505","B506", "B323", "B610", "B703", "B611", "B412","B413"]
 exclude_dirs = ["./test/*"]
+
+[tool.pylint.main]
+exit-zero = true
+jobs = 0
+recursive = true
+suggestion-mode = true
+# ignore-paths =
+
+[tool.pylint.basic]
+class-attribute-naming-style = "snake_case"
+
+[tool.pylint.classes]
+defining-attr-methods = ["__init__", "__new__", "__post_init__"]
+
+[tool.pylint.imports]
+# Deprecated modules which should not be used, separated by a comma.
+# deprecated-modules =
+
+# Force import order to recognize a module as part of the standard compatibility
+# libraries.
+# known-standard-library =
+
+[tool.pylint.logging]
+logging-format-style = "new"
+
+disable = [
+    "raw-checker-failed",
+    "bad-inline-option",
+    "locally-disabled",
+    "file-ignored",
+    "suppressed-message",
+    "useless-suppression",
+    "deprecated-pragma",
+    "use-symbolic-message-instead",
+    "design",
+    "line-too-long",
+    "too-many-lines",
+    "too-many-nested-blocks",
+    "typecheck",
+    "import-error",
+    "trailing-whitespace"
```

Comment:
I think removing trailing whitespaces is important, is it checked by another linter (I couldn't find) ? If yes it's ok.

Reply:
As said in the PR message yes it is done by another package. There is a pre-commit hook that runs autopep8 which should take care of this.